content/rs/administering/database-operations/slave-ha.md

```diff
@@ -84,7 +84,7 @@ Slave HA migrates slave shards for databases according to this order of priority
 rladmin tune db <bdb_uid> slave_ha_priority <positive integer>
     ```
 
-1. CRDBs - The CRDB synchronization uses slave shards to synchronize between the replicas.
+1. Active-Active databases - The Active-Active database synchronization uses slave shards to synchronize between the replicas.
```

Comment:
s/The Active-Active database synchronization/Active-Active database synchronization/